src/state/slices/opportunitiesSlice/resolvers/idle/index.ts

```diff
@@ -88,13 +88,15 @@ export const idleStakingOpportunitiesMetadataResolver = async ({
           tvl: opportunity.tvl.balanceUsdc.toFixed(),
           name: `${underlyingAsset.symbol} Vault`,
           version: opportunity.version,
+          provider: DefiProvider.Idle as const,
```

Comment:
`as const` will be required for the stacked PR, so might as well while diffing this